libcontainer/specconv/spec_linux.go

```diff
@@ -184,6 +209,30 @@ func KnownMountOptions() []string {
 	return res
 }
 
+// KnownMemoryPolicyModes returns the list of the known memory policy modes.
+// Used by `runc features`.
```

Comment:
nit: please don't use markdown in docstrings, it's not supported.

Reply:
ok.